dev/update_mlflow_versions.py

```diff
@@ -8,8 +8,11 @@
 
 def get_current_py_version() -> str:
     text = Path("mlflow", "version.py").read_text()
-    ver = Version(re.search(r'VERSION = "(.+)"', text).group(1))
-    return f"{ver.major}.{ver.minor}.{ver.micro}"
+    return re.search(r'VERSION = "(.+)"', text).group(1)
```

Comment:
To avoid confusion, do not strip the prefix and just return the current mlflow version for Python.